src/csvcubed/models/validationerror.py

```diff
@@ -60,3 +60,11 @@ class PydanticThrowableSpecificValidationError(
     """
 
     pass
+
+@dataclass
+class ValidateModelProperiesError(ValidationError):
+
+    """
+    This error will be returned with the name of the class attribute.
+    """
+    property_name: str
```

Comment:
Can you create a ticket to expand the information we hold here? I think it'd be useful to include a path to the property instead of just a single property_name.

Reply:
Isn't that provided in the script that utilises these functions? `pref_check.py`

` print(
                f"In {j.__name__} ({inspect.getfile(j)}) you have not validated {', '.join(not_validated)}"
            )
            print(format(field_names))  # This returns empty`

this will print out the variable names that need validation and the file path.  but i can of course make a ticket if it's still required.

Reply:
for example what data should be stored in that class.
```suggestion
    property_path: List[str]
```

e.g. if we have the following:

```json
{
  "chicken": {
    "burgers": [
      {
        "should_be_a_string": 4
      },
      {
        "should_be_a_string": "woof"
      }
    ]
  }
}
```

The property path for the offending validation error should be ["chicken", "burgers", "0", "should_be_a_string"]